calendar-rest-api/src/main/java/com/linagora/calendar/restapi/auth/BasicAuthenticationStrategy.java

```diff
@@ -92,6 +95,10 @@ public Mono<MailboxSession> createMailboxSession(HttpServerRequest httpRequest)
     }
 
     private Mono<Username> authenticate(UserCredentials creds) {
+        if (creds.username.asString().equals(restApiConfiguration.getAdminUsername())
```

Comment:
What do you think about using a single username/password in the userDAO/userRepository as admin credentials?
If Sabre only needs one credential to query email users, this is enough.